packages/neos-ui-redux-store/src/UI/Drawer/index.spec.js

```diff
@@ -68,3 +70,34 @@ test(`The "hide" action should set the "isHidden" key to "true".`, () => {
     expect(nextState1.get('ui').get('drawer').get('isHidden')).toBe(true);
     expect(nextState2.get('ui').get('drawer').get('isHidden')).toBe(true);
 });
+
+test(`The reducer should initially mark the menu group "content" as collapsed.`, () => {
+    const state = new Map({});
+    const nextState = reducer(state, {
+        type: system.INIT
+    });
+
+    expect(nextState.get('ui').get('drawer').get('collapsedMenuGroups').includes('content')).toBe(true);
+});
+
+test(`The "toggleMenuGroup" action should add or remove given group from "collapsedMenuGroups".`, () => {
+    const state = Immutable.fromJS({
+        ui: {
+            drawer: {
+                collapsedMenuGroups: []
+            }
+        }
+    });
+    const nextState1 = reducer(state, actions.toggleMenuGroup('content'));
+    const nextState2 = reducer(nextState1, actions.toggleMenuGroup('management'));
+    const nextState3 = reducer(nextState2, actions.toggleMenuGroup('content'));
+    const nextState4 = reducer(nextState3, actions.toggleMenuGroup('management'));
+
+    expect(nextState1.get('ui').get('drawer').get('collapsedMenuGroups').includes('content')).toBe(true);
+    expect(nextState2.get('ui').get('drawer').get('collapsedMenuGroups').includes('content')).toBe(true);
+    expect(nextState2.get('ui').get('drawer').get('collapsedMenuGroups').includes('management')).toBe(true);
+    expect(nextState3.get('ui').get('drawer').get('collapsedMenuGroups').includes('content')).toBe(false);
+    expect(nextState3.get('ui').get('drawer').get('collapsedMenuGroups').includes('management')).toBe(true);
+    expect(nextState4.get('ui').get('drawer').get('collapsedMenuGroups').includes('content')).toBe(false);
+    expect(nextState4.get('ui').get('drawer').get('collapsedMenuGroups').includes('management')).toBe(false);
+});
```

Comment:
not sure if these actually pass, running `make test` passes but doesn't seem to run the actual tests?

Reply:
There is a similar case where tests don't seem to run correctly.

Reply:
So, I guess we need to fix this before we can merge this PR, right? Any idea what might be the problem?

Reply:
No idea, never ran the tests previously so no idea how it's supposed to work.